src/platform/config/CMakeLists.txt

```diff
@@ -8,8 +8,20 @@ project(config)
 
 if(APPLE)
 
-  find_library(foundation_config NAMES Foundation REQUIRED)
-  message("Foundation found at ${foundation_config}.")
+    #find_library(foundation_config NAMES Foundation REQUIRED)
+    #message("Foundation found at ${foundation_config}.")
+
+  # We need to get the proper SDK paths in the build
+  execute_process(
```

Comment:
This is repetitive with logic in the top level CMakeLists.txt. We should keep the SDK path lookup in a single place, and use a variable to refer to it. We also should keep macOS/iOS framework search paths consistent (such that if we're doing an iOS build, it should use iOS SDK path everywhere).